### PR TITLE
Fix example code for use on OSX

### PR DESCRIPTION
**Pull request by mmichal9**

### CODE REVIEW

File: .gitignore [hunk 1]
```diff
@@ -0,0 +1,16 @@
...
+.DS_Store
+.DS_Store?
+._*
+.Spotlight-V100
+.Trashes
+ehthumbs.db
+Thumbs.db
+
+# Example build files
+examples/basic/basic
```
Comment (deadprogram) [inline .gitignore:14]: Not sure where these directories come from? I do not see how that would be created by anything *currently* in the project.

**Comment by deadprogram** [comment]
Looks at this again, I think if you just passed in the string "0" it *should* work on macOS. Can you try that out, please?

File: .gitignore (hunk 1 above)
Comment (mmichal9) [reply .gitignore:14]: This is just in case you use `go build .` instead of `go run`.

**Comment by mmichal9** [comment]
`gocv.OpenVideoCapture("0")` works ok on OSX. Do you know if this also works on Windows / Linux ?
 Otherwise, I suppose OS could be checked with `runtime.GOOS` on the program start and correct device string selected.

**Comment by deadprogram** [comment]
> Do you know if this also works on Windows / Linux ?

It does work for sure on Linux, and AFAIK on Windows as well.

**Comment by mmichal9** [comment]
@deadprogram I have made some changes to switch back to gocv.OpenVideoCapture() with "0" as a parameter.

File: examples/wasmcam/main.go [hunk 2]
```diff
@@ -22,7 +22,7 @@ import (
 var processFrameWasm []byte
 
 var (
-	device = flag.String("device", "/dev/video0", "video capture device")
+	device = flag.String("device", "0", ":video capture device")
```
Comment (deadprogram) [inline examples/wasmcam/main.go]:
```suggestion
	device = flag.String("device", "0", "video capture device")
```

I think this is just an extra character that slipped in?

Comment (mmichal9) [reply examples/wasmcam/main.go]: yep, should be fixed now :)

**Comment by deadprogram** [comment]
Thank you very much for the corrections here @mmichal9 now squash/merging.

**Review by deadprogram — APPROVED** [review]

**Pull request merged**